src/App.tsx

```diff
@@ -15,48 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum Classes {
+  rev = 'reverse',
+  alphabet = 'alphabet',
+  alphabetRev = 'alphabet reverse',
+  length = 'length',
+  lengthRev = 'length reverse',
```

Comment:
You don't need separate values for reversed values. Refactor the code to avoid it 
```suggestion
  length = 'length',
```